### ISSUE TITLE
Macbert训练结果偏差很大

### ISSUE THREAD

**Issue by xdnjust** [shibing624/pycorrector#288]
楼主大神你好，感谢你的开源贡献！

我重新训练macbert，最后在Sighan15 test测试集上得到的sentence P、R、F1分别只有0.64、0.34、0.45，和你给出的结果相差很大
想问一下是不是训练出现问题，因为验证集的detection_acc和correction_acc也只有0.63和0.56

我的训练设置：
训练数据集：SIGHAN+Wang271K中文纠错数据集
预训练模型：chinese-macbert-base，
loss_weight = 0.7
train_batch_size = 64
dev_batch_size = 32
test_batch_size = 1
learning_rate = 4e-5
weight_decay = 0.0
adam_epsilon = 1e-8
max_grad_norm = 1.0
num_train_epochs = 30
warmup_ratio = 0.1
seed = 1111
early_stop = 10

**Comment by shibing624** [comment]
数据shuf了吗？用默认参数试下。

**Comment by xdnjust** [comment]
已解决，是预训练模型参数未导入成功的原因
